src/Benchmark.jl

```diff
@@ -0,0 +1,17 @@
+include("Solution.jl")
+
+# localSearch should obey a certain interface. 
```

Comment:
Apparently there is no way to implement interfaces in Julia, for now, we could do localSearch::Tuple{Swap, TwoOpt}